CHANGELOG.md

```diff
@@ -62,6 +59,17 @@ Main (unreleased)
 
 - Add `extra_labels` configuration to eventhandler integration. (@hjet)
 
+v0.24.2 (2022-05-02)
+--------------------
+
+### Bugfixes
+
+- Added config watcher delay to prevent race condition in cases where scraping service mode has not gracefully exited. (@mattdurham)
```

Comment:
```suggestion
- Added configuration watcher delay to prevent race condition in cases where scraping service mode has not gracefully exited. (@mattdurham)
```

Reply:
Is there any other way of saying "gracefully exited"? I am not sure what it means..

Reply:
It's frequently used as a technical term. Its meaning usually is that the process has exited correctly after receiving a signal or finishing its work, by taking care of all its dependencies, cleanup tasks, notifying its peers etc..

The opposite would be a process that was violently killed without having the time to communicate it properly (like in cases where it ran out of memory or disk space, or when the host machine was shut down unexpectedly).